community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelTransactionImplementationHandle.java

```diff
@@ -102,6 +104,13 @@ public boolean isUnderlyingTransaction( KernelTransaction tx )
     }
 
     @Override
+    public Stream<ExecutingQuery> executingQueries()
+    {
+        KernelStatement statement = tx.tryAcquireStatement();
```

Comment:
From what I can see, if we fail to acquire a statement, we will get `null` back, which we handle.

Regarding reused transactions - how can we protect against this?